src/index.html

```diff
@@ -1,13 +1,406 @@
+                  alt="Max Henry"
+                  class="testimonial__photo"
+                  >
+                </div>
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">Max Henry</a>
+                </h3>
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-ua" id="contact-ua">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/RGvTEg2yzS3j1f1y7"
+                  class="contact-us__info-link">
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contact-us__field form-field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us__feild contact-us__feild--last form-field form-field--textarea">
+                </textarea>
```

Comment:
```suggestion
                <textarea
                  name="message"
                  placeholder="Message"
                  class="contact-us__feild contact-us__feild--last form-field form-field--textarea"
               ></textarea>
```
Існує баг, коли в текстарев не відображається плейсхолдер, а текст має дивні відступи. Це фікситься от таким от форматуванням, як я вказав в прикладі. (в одну лінію і один відступ вліво)
Також давай зробимо всі поля required, бо зараз можна відправити пусту форму

<img width="476" alt="Снимок экрана 2021-10-11 в 10 44 23" src="https://user-images.githubusercontent.com/66570142/136751856-8d2080a6-9511-45f0-b5db-08cea4babb03.png">
<img width="476" alt="Снимок экрана 2021-10-11 в 10 44 27" src="https://user-images.githubusercontent.com/66570142/136751861-7a825088-88a3-416f-b418-7c2e02ee5221.png">

Reply:
Дякую! Виправив)